docs/usage/esql.asciidoc

```diff
@@ -0,0 +1,34 @@
+[[esql]]
+=== ES|QL in the Java client
+
+This page enables you to understand and use {ref}/esql.html[ES|QL] utilizing the
+Java client.
+
+
+[discrete]
+[[esql-how-to]]
+==== How to use the ES|QL API
+
+[TO DO]
+
+// This section describes the use of the "raw" ES|QL API.
+
+
+[discrete]
+[[esql-consume-results]]
+==== Consuming ES|QL results
```

Comment:
Should "Consuming" be "Consume" instead, according to our [WIP style guidelines about gerunds in headings](https://docs.google.com/document/d/1LXtIbBP4f64IlivFSorPqxFzSa0cwpfRDy7-2zUkgSw/edit#heading=h.silftn4u1x6v)? Not sure.

Applies to the "Defining" heading on line 29 too.